.script/tests/KqlvalidationsTests/KqlValidationTests.cs

```diff
@@ -1,164 +1,207 @@
-            if (!validationResult.IsValid)
-            {
-                firstErrorLocation = GetLocationInQuery(queryStr, validationResult.Diagnostics.First(d => d.Severity == "Error").Start);
-            }
-
-            var listOfDiagnostics = validationResult.Diagnostics;
-
-            bool isQueryValid = !(from p in listOfDiagnostics
-                               where !p.Message.Contains("_GetWatchlist") //We do not validate the getWatchList, since the result schema is not known
-                               select p).Any();
-
-
-            Assert.True(
-                isQueryValid,
-                isQueryValid
-                    ? string.Empty
-                    : @$"Template Id: {id} is not valid in Line: {firstErrorLocation.Line} col: {firstErrorLocation.Col}
-                    Errors: {validationResult.Diagnostics.Select(d => d.ToString()).ToList().Aggregate((s1, s2) => s1 + "," + s2)}");
-        }
-
-        private Dictionary<object, object> ReadAndDeserializeYaml(string encodedFilePath)
-        {
-        
-            var yaml = File.ReadAllText(Utils.DecodeBase64(encodedFilePath));
-            var deserializer = new DeserializerBuilder().Build();
-            return deserializer.Deserialize<dynamic>(yaml);
-        }
-        private bool ShouldSkipTemplateValidation(string templateId)
-        {
-            return TemplatesToSkipValidationReader.WhiteListTemplates
-                .Where(template => template.id == templateId)
-                .Where(template => !string.IsNullOrWhiteSpace(template.validationFailReason))
-                .Where(template => !string.IsNullOrWhiteSpace(template.templateName))
-                .Any();
-        }
-
-        private (int Line, int Col) GetLocationInQuery(string queryStr, int pos)
-        {
-            var lines = Regex.Split(queryStr, "\n");
-            var curlineIndex = 0;
-            var curPos = 0;
-
-            while (lines.Length > curlineIndex && pos > curPos + lines[curlineIndex].Length + 1)
-            {
-                curPos += lines[curlineIndex].Length + 1;
-                curlineIndex++;
-            }
-            var col = (pos - curPos + 1);
-            return (curlineIndex + 1, col);
-        }
-    }
-
-}
-
+        private const int TestFolderDepth = 3;
+
+        public KqlValidationTests()
+        {
+            _queryValidator = new KqlQueryAnalyzerBuilder()
+               .WithSentinelDefaultTablesAndFunctionsSchemas()
+               .WithCustomTableSchemasLoader(new CustomJsonDirectoryTablesLoader(Path.Combine(Utils.GetTestDirectory(TestFolderDepth), "CustomTables")))
+               .WithCustomFunctionSchemasLoader(new CustomJsonDirectoryFunctionsLoader(Path.Combine(Utils.GetTestDirectory(TestFolderDepth), "CustomFunctions")))
+               .WithCustomFunctionSchemasLoader(new ParsersCustomJsonDirectoryFunctionsLoader(Path.Combine(Utils.GetTestDirectory(TestFolderDepth), "CustomFunctions")))
+               .Build();

```

Comment:
It is difficult to see the diff since the whole formatting changed. Can you maybe explain the changes in the file / guide me to the lines that were changed?

Reply:
![image](https://user-images.githubusercontent.com/46525261/162693000-81ea29f3-8c2b-4247-9fdf-384699e5df64.png)

Can you try to hide the whitespace? 
The new lines are 25, 110-130, 185-203

Sorry for the inconvenience.

Reply:
Thanks!